client/rpcserver/handlers.go

```diff
@@ -301,86 +308,108 @@ func sortHelpKeys() []string {
 	return keys
 }
 
-// helpMsgs are a map of routes to help messages. The first string is a short
-// description of usage. The second is a brief description and a breakdown of
-// the arguments and return values.
-var helpMsgs = map[string][2]string{
-	helpRoute: {`("cmd")`,
-		`Print a help message.
-
-Args:
+// helpMsgs are a map of routes to help messages. They are broken down into six
+// sections.
+// In descending order:
+// 1. Password argument example inputs. These are arguments the caller may not
+//    want to echo listed in order of input.
+// 2. Argument example inputs. These are non-sensitive arguments listed in order
+//    of input.
+// 3. A description of the command.
+// 4. An extensive breakdown of the password arguments.
+// 5. An extensive breakdown of the arguements.
+// 6. An extensive breakdown of the returned values.
+var helpMsgs = map[string][6]string{
+	helpRoute: {
+		``,                           // password args example input
+		`("cmd") (includePasswords)`, // args example input
+		`Print a help message.`,      // command explanation
+		``,                           // password args breakdown
+		`Args:
     cmd (string): Optional. The command to print help for.
-
-Returns:
-    string: The help message for command.`,
+    includePasswords (bool): Optional. Default is false. Whether to include
+      password arguments in the returned help.`, // args breakdown
+		`Returns:
+    string: The help message for command.`, // returns breakdown
 	},
-	versionRoute: {``,
-		`Print the dex client rpcserver version.
-
-Returns:
+	versionRoute: {
+		``,
+		``,
+		`Print the dex client rpcserver version.`,
+		``,
+		``,
+		`Returns:
     string: The dex client rpcserver version.`,
 	},
-	initRoute: {`"appPass"`,
-		`Initialize the client.
-
-Args:
-    appPass (string): The dex client password.
-
-Returns:
+	initRoute: {
+		`"appPass"`,
+		``,
+		`Initialize the client.`,
+		`Password Args:
+    appPass (string): The dex client password.`,
+		``,
+		`Returns:
     string: The message "` + initializedStr + `"`,
 	},
-	preRegisterRoute: {`"dex"`,
-		`Preregister for dex.
-
-Args:
-    url (string): The dex to preregister for.
-    cert-path (string): Optional. The filepath to the TLS certificate for the dex.
-
-Returns:
+	preRegisterRoute: {
+		``,
+		`"dex" ("cert")`,
+		`Preregister for dex.`,
+		``,
+		`Args:
+    dex (string): The dex address to preregister for.
+    cert (string): Optional. The TLS certificate for the dex. For dexcctl, the
+      certificate's path should be used.`,
```

Comment:
Use the old description. The user doesn't provide the certificate, they provide the file path, and they are always using **dexcctl**.